### PR TITLE
Pulseaudio for sound tests

### PR DESCRIPTION
**Pull request by allandequeiroz**
I've put some comments here #73 
Thanks!

### CODE REVIEW

**Comment by gdams** [comment]
rather than doing a regex replace you should be able to do something like this I think:
``` yaml
- name: Adding existing user jenkins to group audio
  user: name=jenkins
        groups=audio
        append=yes
```

**Comment by allandequeiroz** [comment]
Hi @gdams thanks a lot for let me know about this way. Just changed.

File: Build/ansible/playbooks/ubuntu_14_16_x86.yml [hunk 1]
```diff
@@ -172,6 +172,14 @@
     - include: slack_notification/successful_slack.yml
       static: yes
 
+    - name: Include Jenkins to audio group
```
Comment (gdams) [inline Build/ansible/playbooks/ubuntu_14_16_x86.yml]: nit: Include Jenkins to audio group ==> Add jenkins to the audio group

Comment (allandequeiroz) [inline Build/ansible/playbooks/ubuntu_14_16_x86.yml]: 👍  Sorry about that, I'll change that tonight when I get back to home.

**Comment by allandequeiroz** [comment]
Hi there. I've just updated.

Let me know if we should group "apts" in a different way,
we sort of did it for Test and Build tools in another issue.

Maybe we could put everything together under the same task and
Include comments with the former tasks names just to explain
what those packages are related to.

**Comment by karianna** [comment]
@bblondin OK to land this now?

**Comment by bblondin** [comment]
@karianna I'm ok with it...

**Pull request merged**